src/plugins/intel_gna/tests/functional/pass_tests/concat_restrictions.cpp

```diff
@@ -333,6 +411,14 @@ TEST_P(ConvConcatNHWCRestrictionsPos, CompareWithRefImpl) {
     Run();
 };
 
+TEST_P(ConvConcatConcatNHWCRestrictionsNeg, CompareWithRefImpl) {
+    ExpectLoadNetworkToThrow(getMatch());
+};
+
+TEST_P(ConvConcatConcatNHWCRestrictionsPos, CompareWithRefImpl) {
+    Run();
+};
+
 const std::vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::Precision::FP32};
 const std::vector<std::map<std::string, std::string>> configs = {{{"GNA_DEVICE_MODE", "GNA_SW_FP32"}}};
```

Comment:
maybe it would be good to add GNA_2_0, GNA_3_0 and maybe GNA_3_5 exec target??

Reply:
Sure. I suggest to do this for all tests in a separate PR.

Reply:
the way suggested by @sirzabek is worth of consideration IMO - the concern of NOT adding explicit targets on per test basis was earlier raised in other PR's.